docs/pages/getting-started.mdx

```diff
@@ -14,7 +14,17 @@ layout: tocless-doc
   <Tile icon="kubernetes" title="Kubernetes" href="./kubernetes-access/getting-started/cluster.mdx">
     Get started with Teleport and Kubernetes.
   </Tile>
+  <Tile icon="database" title="Databases" href="./kubernetes-access/getting-started/cluster.mdx">
+    Get started with Teleport and Databases.
+  </Tile>
+  <Tile icon="desktop" title="Desktops" href="./desktop-access/getting-started.mdx">
+    Get started with Teleport and Windows Desktops.
+  </Tile>
+  <Tile icon="windows" title="Applications" href="./application-access/getting-started.mdx">
+    Get started with Teleport and Applications.
+  </Tile>
```

Comment:
I think the repetition of "Get started" in most of these tiles might be a bit noisy. What would you think of this?
1. Adding a section heading titled "Start here" or something similar to this page and include the Linux and Docker guides there, plus the cloud free trial tile. First-timers will be encouraged to click these links.
2. Add a second heading titled "Connect your infrastructure" (or something like that) and include the tiles for Kubernetes, Databases, Desktops, and Applications.
3. We can then remove the "Get started..." caption from the tiles for Kubernetes, Databases, Desktops, and Applications. If we do this, maybe we can change "Desktops" to "Windows Desktops"?

Reply:
I like this. I'll try to update the PR. 

It would also be good if we could make the Terraform & Cloudformation guides front and center. 

-> Start here
     - Manually
     - AWS
         - Terraform ( AWS )  
         - Cloudformation
     - Kubernetes

I still think it's good to have 'Getting started' per protocol, but since 'getting-started' gets a lot of traffic this should make the jumping off spot easier. 

I'll update my PR.

Reply:
I'm also going to keep 'Getting Started' over 'start here', since it already has a lot of traffic and is a well known place to start.. but I'll work on the 2nd heading.